openpype/hosts/houdini/plugins/create/create_composite.py

```diff
@@ -1,44 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating composite sequences."""
 from openpype.hosts.houdini.api import plugin
+from openpype.pipeline import CreatedInstance
 
 
-class CreateCompositeSequence(plugin.Creator):
+class CreateCompositeSequence(plugin.HoudiniCreator):
     """Composite ROP to Image Sequence"""
 
+    identifier = "io.openpype.creators.houdini.imagesequence"
     label = "Composite (Image Sequence)"
     family = "imagesequence"
     icon = "gears"
 
-    def __init__(self, *args, **kwargs):
-        super(CreateCompositeSequence, self).__init__(*args, **kwargs)
+    def create(self, subset_name, instance_data, pre_create_data):
+        import hou  # noqa
 
-        # Remove the active, we are checking the bypass flag of the nodes
-        self.data.pop("active", None)
+        instance_data.pop("active", None)
```

Comment:
Is there a reason to expect `"active"` key in instance data?

Reply:
That is part of the old creator stuff. Active was reflecting the state of the Houdini node but that can be handled better now.